cosmos/providers/dbt/core/operators/local.py

```diff
@@ -156,7 +159,7 @@ def run_command(
 
             logger.info(f"Trying to run the command:\n {cmd}\nFrom {tmp_project_dir}")
```

Comment:
WDYT about moving this logging line to the `run_subprocess` method so it's consistently logged?

Reply:
That's a good suggestion, @josh-fell !

I started making this change, @josh-fell, but that would require extending the named arguments on both the `local` and `virtualenv` `BaseOperators` `run_subprocess` methods and adding logging logic to this method in both classes - so it felt we were adding more complexity than needed. Moving forward, as part of another PR, we can remove this log message and change the logging in the `subprocess_hook.run_command`.